grama/models/cantilever_beam.py

```diff
@@ -1,11 +1,11 @@
 __all__ = ["make_cantilever_beam"]
 
-import numpy as np
 import grama as gr
```

Comment:
It seems ok to import the full Grama package within model building, but perhaps we should document this practice somewhere....